src/umocktypes_wcharptr.c

```diff
@@ -12,6 +12,14 @@
 #include "umock_c/umockalloc.h"
 #include "umock_c/umock_log.h"
 
+void static mbstate_t_init(mbstate_t* ps)
+{
+    (void)memset(ps, 0, sizeof(ps));
+    const wchar_t empty[] = L"";
+    const wchar_t* srcp = empty;
+    wcsrtombs(NULL, &srcp, 0, ps);
```

Comment:
>wcsrtombs [](start = 4, length = 9)

(void) since we're ignoring the result on purpose.

Reply:
void added